python/orca/example/learn/pytorch/brainMRI/dataset.py

```diff
@@ -0,0 +1,82 @@
+import os
+from os.path import exists
+from os import makedirs
+import argparse
+import matplotlib.pyplot as plt
+import numpy as np
+import glob
+import random
+import cv2
+from PIL import Image
+import pandas as pd
+
+import torch
+import torchvision
+import torchvision.transforms as transforms
+from torchvision.utils import make_grid
+import torchvision.transforms as tt
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+import torch.utils.data as data
+from torch.utils.data import DataLoader
+
+from mpl_toolkits.axes_grid1 import ImageGrid
+import albumentations as A
+from sklearn.model_selection import train_test_split
+from tqdm import tqdm
+
+def set_seed(seed = 0):
+    torch.backends.cudnn.deterministic = True
+    torch.backends.cudnn.benchmark = False
+    torch.manual_seed(seed)
+    torch.cuda.manual_seed_all(seed)
+    np.random.seed(seed)
+    random.seed(seed)
+
+def diagnosis(mask_path):
+    return 1 if np.max(cv2.imread(mask_path)) > 0 else 0
+
+def dataset():
+    ROOT_PATH = '/home/mingxuan/BigDL/python/orca/example/learn/pytorch/brainMRI/data/kaggle_3m/'
+    mask_files = glob.glob(ROOT_PATH + '*/*_mask*')
+    image_files = [file.replace('_mask', '') for file in mask_files]
+    files_df = pd.DataFrame({"image_path": image_files,
+                    "mask_path": mask_files,
+                    "diagnosis": [diagnosis(x) for x in mask_files]})
+
+    train_df, test_df = train_test_split(files_df, stratify=files_df['diagnosis'], test_size=0.15, random_state=0)
+    train_df = train_df.reset_index(drop=True)
+    test_df = test_df.reset_index(drop=True)
+
+    return train_df, test_df
```

Comment:
Put this in brainMRI.py? Keep the unet file and dataset file unchanged as the original one?